apps/cyberstorm-remix/app/p/components/ReportPackage/ReportPackage.tsx

```diff
@@ -147,22 +168,91 @@ export function ReportPackageForm(props: ReportPackageFormFullProps) {
             rootClasses="report-package__textarea"
           />
         </div>
-        {error && (
+        {strongForm.inputErrors ||
+        strongForm.refineError ||
+        strongForm.submitError ? (
           <div className="report-package__block">
-            <NewAlert csVariant="danger">{error}</NewAlert>
+            {strongForm.inputErrors ? (
+              <NewAlert csVariant="danger">
+                {Object.entries(strongForm.inputErrors).map(([key, value]) => (
+                  <div key={key}>
+                    <strong>{key}:</strong>{" "}
+                    {Array.isArray(value) ? value.join(", ") : value}
+                  </div>
+                ))}
+              </NewAlert>
+            ) : null}
+            {strongForm.refineError ? (
+              <NewAlert csVariant="danger">
+                Error while refining: {strongForm.refineError.message}
```

Comment:
IDK what refining means and I assume neither does the users.

Reply:
> Refining the form data is done when the input data doesn't match the wanted data format of the endpoint. 

Thanks for the clarification, but telling this to *me* doesn't help our less tech-savvy users when they encounter the error message in production. What sort of error messages can we expect from `strongForm.refineError.message`? If they're human friendly we can probably drop the whole "Error while refining:" part? If not, would something like "Form contains invalid values:" work in combination the `strongForm.refineError.message`?